workflow/resources/datavzrd/diffexp-template.yaml

```diff
@@ -0,0 +1,280 @@
+name: ?f"Differential expression analysis for model {wildcards.model}"
+datasets:
+  genes_representative:
+    path: ?input.genes_representative
+    links:
+      link to transcripts:
+        column: ens_gene
+        table-row: transcripts/ens_gene
+      link to log-count matrix:
+        column: target_id
+        table-row : logcount_matrix/transcript
+      link to genes representative:
+        column: ens_gene
+        table-row: genes_representative/ens_gene
+      link to volcano plot:
+        column: ens_gene
+        view: volcano-plots
+    separator: "\t"
+  transcripts:
+    path: ?input.transcripts
+    links:
+      link to genes representative:
+        column: ens_gene
+        table-row: genes_representative/ens_gene
+    separator: "\t"  
+  genes_aggregated:
+    path: ?input.genes_aggregated
+    separator: "\t"
+  logcount_matrix:
+    path: ?input.logcount_matrix
+    links:  
+      link to transcripts:
+        column: transcript
+        table-row: transcripts/target_id
+    separator: "\t"
+default-view: genes_representative
+views:
+  genes_representative:
+    dataset: genes_representative
+    desc: |
+      Differentially expressed genes using the model computed with sleuth by taking only a manually configured representative transcript for each gene.
+      The columns `b_*` and `b_*_se` display the effect size β and the corresponding standard error for every covariate in the model. They are analog to log2 fold changes. Note that non-binary covariates are binarized by sleuth, leading to multiple columns `b_*1`, `b_*2`, etc.
+      In addition, a signed version of the pi-value score (as proposed by `Xiao et al. 2014 <https://dx.doi.org/10.1093/bioinformatics/`) is shown. The sign reflects the sign of the effect (i.e. positive for upregulation, negative for downregulation).
+    page-size: 25
+    render-table:
+      columns:
+        ens_gene:
+          display-mode: hidden       
+        ext_gene:
+          display-mode: normal
+        gene_desc:
+          display-mode: normal
+        canonical:
+          display-mode: normal
+          plot:
+            heatmap:
+              scale: ordinal
+              domain:
+                - TRUE
+                - FALSE
+              range:
+                - "#aec7e8"
+                - white
+              custom-content: |
+                function(value, row) {
+                    if(value == "TRUE") {
+                        return "yes"
+                    } else {
+                        return ""
+                    }
+                }
+        target_id:
+          display-mode: normal
+          link-to-url: "http://www.ensembl.org/Homo_sapiens/Gene/Summary?g={target_id}"
+        pval:
+          plot:
+            bars:
+              scale: linear
+              domain:
+                - 0.0000000000000000000000000000000000000000000000000000000000000000001
```

Comment:
Does scientific notation not work here? Also, why not just from 0.0 to 1.0?

Reply:
This is now updated. I guess in the previous version, there was an issue with the bar scale.